apps/withdraw-pool/src/utils/decodeExitPoolHookCalldata.ts

```diff
@@ -28,7 +28,7 @@ export async function decodeExitPoolHookCalldata(
 
   const calls = decodedFunctionData.args[0] as ICalldata[];
 
-  const lastCallIndex = calls.length - 1;
+  const lastCallIndex = calls.length - 3;
```

Comment:
would add a comment to explain this, I mean, why is lastCall specifically equals to length - 3?